linetools/analysis/interactive_plot.py

```diff
@@ -490,6 +491,23 @@ def on_keypress(self, event):
             self.contpoints[ind] = event.xdata, event.ydata
             self.contpoints.sort()
             self.update()
+        elif event.key == 'M':
+            if not between(event.xdata, self.wmin, self.wmax):
```

Comment:
We may also need to add a check that len(self.contpoints) > 0

(I forgot about this)